### ISSUE TITLE
UniqueProperties check should allow a unique check across all properties files

### ISSUE THREAD

**Issue by scrapmac** [checkstyle/checkstyle#2779]
Rational: Sometimes, the codebase provides a global helper class to get properties' values. This class reads all properties files from a directory. In such cases, duplicate keys across files will clash with each other.

Although I think that most users might actually just want to limit this option across a single directory. i.e. Check for unique keys in all property files which reside in the same folder.

**Comment by romani** [comment]
for now we have a general model to do validation only with one file , so I renamed your request to new Check request. Current UniqueProperties Check works as designed .

**Comment by scrapmac** [comment]
Wow, this is really fast :) New check also works. Any thoughts on the "per directory" option?

**Comment by romani** [comment]
good Idea, I will think about it when we have time to make this Check. 
Right now we are at moratorium period for new Checks - https://groups.google.com/forum/#!topic/checkstyle-devel/JbhfiiSqKsQ.
if you need this Check you can make it yourself http://checkstyle.sourceforge.net/writingchecks.html and contribute to sevntu-checkstyle project as it get maturity we will move it to main repo.

**Comment by scrapmac** [comment]
If the check development has not started, then what is the commit mentioned in this issue for then?

**Comment by romani** [comment]
sorry, I was a typo in reference of issue. @sabaka will fix this typo.
No body is started to implement it. 
If you wish to start please open the same issue at https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues and start implementation , you are welcome with any questions.
